src/components/IngredientFeed.vue

```diff
@@ -1,14 +1,108 @@
 <template>
   <div class="container">
-    <header class="jumbotron">
-      <h3>This is the ingredients page</h3>
-    </header>
-
     <!-- Example of how to use MDB rows to open the centered modal view -->
     <!-- <MDBRow center class="text-center">
       <h3>Vue Teleport Example</h3>
       <ModalView />
     </MDBRow> -->
+    <div class="tool-bar">
+      <div class="tool-bar-container">
+        <button type="button" class="btn">Sort By</button>
+      </div>
+      <div class="tool-bar-container">
+        <div class="form-outline">
+          <input
+            type="search"
+            id="form1"
+            class="form-control"
+            placeholder="Search Ingredients"
+            aria-label="Search"
+          />
+        </div>
+        <!-- TODO replace with an actual material icon -->
+        <button type="button" class="btn btn-primary btn-floating">
+          <i class="fas fa-plus"></i>
+        </button>
+      </div>
+    </div>
+    <!-- TODO see if this can be done in a list -->
```

Comment:
We'll need to have this work as a list since we load these items based on the results from the server; let me know if you need help figuring out how to do that